pkg/standalone/stop.go

```diff
@@ -52,7 +53,15 @@ func StopAppsWithRunFile(runTemplatePath string) error {
 	}
 	for _, a := range apps {
 		if a.RunTemplatePath == runTemplatePath {
-			_, err := utils.RunCmdAndWait("kill", fmt.Sprintf("%v", a.CliPID))
+			// Get the process group id of the CLI process.
+			pgid, err := syscall.Getpgid(a.CliPID)
+			if err != nil {
+				// Fall back to cliPID if pgid is not available.
+				_, err = utils.RunCmdAndWait("kill", fmt.Sprintf("%v", a.CliPID))
+				return err
+			}
+			// Kill the whole process group.
+			_, err = utils.RunCmdAndWait("kill", "-SIGINT", "--", fmt.Sprintf("-%v", pgid))
```

Comment:
should `syscall.Kill(-pgid, syscall.SIGINT)` be used instead of this?

Reply:
changed to `syscall.Kill`. Seems both same. Did not notice any difference regarding `cmd.start` and `cmd.wait` in `RunCmdAndWait `.